src/azure-cli/azure/cli/command_modules/role/custom.py

```diff
@@ -1157,7 +1157,7 @@ def list_service_principal_owners(client, identifier):
 def create_service_principal_for_rbac(
         # pylint:disable=too-many-statements,too-many-locals, too-many-branches, unused-argument
         cmd, display_name=None, years=None, create_cert=False, cert=None, scopes=None, role=None,
-        show_auth_for_sdk=None, skip_assignment=False, keyvault=None):
+        show_auth_in_json=None, skip_assignment=False, keyvault=None):
```

Comment:
Internal name change is not customer-facing and shouldn't count as breaking change.